npm-debug.log

```diff
@@ -0,0 +1,605 @@
+0 info it worked if it ends with ok
```

Comment:
You probably didn't want to check this in.  Probably don't even want history of it, so best to rewrite this commit.